sycl/test-e2e/KernelCompiler/kernel_compiler_sycl_jit.cpp

```diff
@@ -612,12 +562,36 @@ int test_warning() {
   return 0;
 }
 
+int test_no_visible_ids() {
+  namespace syclex = sycl::ext::oneapi::experimental;
+  using source_kb = sycl::kernel_bundle<sycl::bundle_state::ext_oneapi_source>;
+  using exe_kb = sycl::kernel_bundle<sycl::bundle_state::executable>;
+  sycl::queue q;
+  sycl::context ctx = q.get_context();
+
+  bool ok =
+      q.get_device().ext_oneapi_can_compile(syclex::source_language::sycl_jit);
+  if (!ok) {
+    return 0;
+  }
+  std::string build_log;
```

Comment:
You can drop the build log to make the test even more compact.

Reply:
Removed it, thank you